serde_at/src/ser/mod.rs

```diff
@@ -426,30 +408,28 @@ where
 }
 
 /// Serializes the given data structure as a string
-pub fn to_string<B, C, T>(
+pub fn to_string<T, const B: usize, const C: usize>(
```

Comment:
Is there a convention to list const generics after generic types? Just curious...

Reply:
It's a requirement